recipes/openff-toolkit/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "openff-toolkit" %}
+{% set version = "0.8.0" %}
```

Comment:
I would have no objection to this package going live whenever you need it -- It has a different name than the `omnia` channel recipe so this won't confuse users or break builds.